<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LecteurAudio</class>
 <widget class="QWidget" name="LecteurAudio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1767</width>
    <height>1058</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(176, 196, 229);</string>
  </property>
  <widget class="QPushButton" name="pushButtonPlay">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>230</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/jouer.png</normaloff>:/images/jouer.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>230</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/icone-arreter.png</normaloff>:/images/icone-arreter.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMusiquePrecedente">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>230</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/icone-precedent.png</normaloff>:/images/icone-precedent.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTitreMusique">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>90</y>
     <width>681</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitreMusique">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>50</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Titre :</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxMarqueur">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>470</y>
     <width>1201</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Boucle</string>
   </property>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>591</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonMarqueur2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/images/icone-marqueur2.png</normaloff>:/images/icone-marqueur2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>75</width>
       <height>75</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTempsMarqueur2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMoins2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/images/icone-moins.png</normaloff>:/images/icone-moins.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPlus2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/images/icone-plus.png</normaloff>:/images/icone-plus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>581</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonPlus1">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>90</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/images/icone-plus.png</normaloff>:/images/icone-plus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMoins1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/images/icone-moins.png</normaloff>:/images/icone-moins.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>80</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTempsMarqueur1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMarqueur1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/images/icone-marqueur1.png</normaloff>:/images/icone-marqueur1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>75</width>
       <height>75</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonMusiqueSuivante">
   <property name="geometry">
    <rect>
     <x>1490</x>
     <y>230</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/icone-suivant.png</normaloff>:/images/icone-suivant.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Volume">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>760</y>
     <width>1201</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Volume</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>501</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonAugmenterVolume">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>130</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/images/icone-volume-plus.png</normaloff>:/images/icone-volume-plus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDiminuerVolume">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/images/icons8-volume-moyen-100.png</normaloff>:/images/icons8-volume-moyen-100.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSliderVolume">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>5</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>671</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="horizontalSliderRapidite">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>521</width>
       <height>71</height>
      </rect>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelLent">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>40</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Lent</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNormal">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Normal</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRapide">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>20</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rapide</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRapiditee">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>80</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="labelNomDuree">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>350</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Durée</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNomTempsCourant">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>350</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Temps Courant</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDuree">
   <property name="geometry">
    <rect>
     <x>1640</x>
     <y>390</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderMusique">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>360</y>
     <width>911</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAvancerSecondes">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>230</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/forward-arrow.png</normaloff>:/images/forward-arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReculerSecondes">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>230</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/reply-arrow.png</normaloff>:/images/reply-arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetListeDeLecture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>481</width>
     <height>1001</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEffacer">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOuvrirMusique">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>40</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/dossier-musique.png</normaloff>:/images/dossier-musique.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBas">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/long-arrow-down.png</normaloff>:/images/long-arrow-down.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHaut">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/long-arrow-up.png</normaloff>:/images/long-arrow-up.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1770</width>
     <height>1061</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(176, 196, 229);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelTempsCourant">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>390</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2:30</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRepeter">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>0</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressources.qrc">
      <normaloff>:/images/repeter.png</normaloff>:/images/repeter.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAleatoire">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>100</y>
      <width>110</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressources.qrc">
      <normaloff>:/images/iconeAleatoireBleue.png</normaloff>:/images/iconeAleatoireBleue.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
  <zorder>frame_5</zorder>
  <zorder>pushButtonPlay</zorder>
  <zorder>pushButtonStop</zorder>
  <zorder>pushButtonMusiquePrecedente</zorder>
  <zorder>lineEditTitreMusique</zorder>
  <zorder>labelTitreMusique</zorder>
  <zorder>groupBoxMarqueur</zorder>
  <zorder>pushButtonMusiqueSuivante</zorder>
  <zorder>groupBox_Volume</zorder>
  <zorder>labelNomDuree</zorder>
  <zorder>labelNomTempsCourant</zorder>
  <zorder>labelDuree</zorder>
  <zorder>horizontalSliderMusique</zorder>
  <zorder>pushButtonAvancerSecondes</zorder>
  <zorder>pushButtonReculerSecondes</zorder>
  <zorder>listWidgetListeDeLecture</zorder>
  <zorder>pushButtonEffacer</zorder>
  <zorder>pushButtonOuvrirMusique</zorder>
  <zorder>pushButtonBas</zorder>
  <zorder>pushButtonHaut</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
